com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -10,16 +10,16 @@ namespace Unity.Netcode
     /// <summary>
     /// Used for local notifications of various scene events.  The <see cref="NetworkSceneManager.OnSceneEvent"/> of
     /// delegate type <see cref="NetworkSceneManager.SceneEventDelegate"/> uses this class to provide
-    /// scene event status.<br/>
-    /// <em>Note: This is only when <see cref="NetworkConfig.EnableSceneManagement"/> is enabled.</em><br/>
-    /// <em>*** Do not start new scene events within scene event notification callbacks.</em><br/>
-    /// See also: <br/>
+    /// scene event status.<br />
```

Comment:
Note: There are several very minor XML API updates I included in this PR which have to do with how our backend XML API documentation is generated and how it deals with <br/> vs <br />... where the former is not recognized and the later is recognized.